NOnion/Directory/NetworkStatusDocument.fs

```diff
@@ -517,3 +517,12 @@ type NetworkStatusDocument =
             self.SharedRandomCurrentValue.Value
         else
             self.SharedRandomPreviousValue.Value
+
+    //HACK: document parser needs to parse ranges but here we use string search instead
+    member self.GetHSDirectories() =
```

Comment:
let's avoid using the HS acronym, use HiddenService

Reply:
Fixed in [39b6655](https://github.com/nblockchain/NOnion/pull/30/commits/39b6655de5c5a956089a5f2049cb6ec8a21e611f)

Reply:
Thing is, commit message still has "hs", but it also lacks the why, which is maybe a hint that this commit should be squashed with another one that makes use of this function.